tests/base.py

```diff
@@ -14,12 +14,25 @@
 import json
 import unittest
 from typing import Optional
+from copy import deepcopy
+from unittest import mock
 
 from requests import Response
 
 from optimizely import optimizely
 
 
+class CopyingMock(mock.MagicMock):
```

Comment:
Is CopyingMock meat as a verb or as a noun?
I wonder if we could have it as a noun, like all classes, just so we have the same flavor to class names?
For example "CopiedMock" or similar?
Thoughts?